### DISCUSSION TITLE
How to search by Euclidian distance using Milvus? [For face search with vectors from CompreFace or Dlib]

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18130] [Q&A and General discussion] [answered]
How to search by Euclidian distance using Milvus? [For face search with vectors from CompreFace or Dlib]

Can someone please share a simple example?

**Comment by yhmo** [comment, marked as answer]
Example of python sdk is here: https://github.com/milvus-io/pymilvus/tree/master/examples

To search with Euclidian distance, just specify the "metric_type" as "L2":

```
    search_param = {
        "data": search_vectors,
        "anns_field": vector_field,
        "param": {"metric_type": "L2", "params": {"nprobe": 10}},
        "limit": 5}
    results = collection.search(**search_param)
```

**Reply by martinenkoEduard** [reply]
Thanks for your great answers! It was really helpful!
Additional question is  Euclidian distance a good choice to select most similar faces (buy embedings that I get from imagenet)?
